src/frontend/src/lib/components/hero/HeroContent.svelte

```diff
@@ -36,34 +34,28 @@
 	class:to-bright-lilac={$networkEthereum}
 >
 	{#if summary}
-		<div
-			transition:slide={SLIDE_PARAMS}
-			class="grid w-full grid-cols-[1fr_auto_1fr] flex-row items-start justify-between"
-		>
-			{#if back}
-				<Back color="current" />
-			{/if}
+		<div transition:slide={SLIDE_PARAMS} class="flex w-full flex-col gap-5">
```

Comment:
Not for this PR, just thinking out at loud and opiniated but, maybe we should spend a bit of time trying other animations. From your video it feels like slide is maybe not the best animation anymore because of the paddings.

Reply:
right! good point! putting it among my backlog and will talk with Stefan too

Reply:
I would just do and to not make a big story of it. Feels trivial. 

"Does the animation looks good? Yes/no. Better like this? Yes. Let's go."

Reply:
This was solved by:

- PR #3221